nebula-unity/Assets/scripts/DungeonMapGenerator.cs

```diff
@@ -118,6 +119,21 @@ public class PotentialDoor {
         maxRoomDiameter = maxDiam;
         shapeFactor = sfact;
         maxAttempts = maxatt;
+=======
```

Comment:
Looks like you have some artifacts here leftover from a stash. (Lines 112, 122, and 136)

Reply:
Actually there are a lot more in GameManager.cs